docs/reference/operators/uidgidresolver.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: UidGidResolver
+sidebar_position: 10
+---
+
+# UidGidResolver
+
+The `UidGidResolver` data operator resolves `uid` and `gid` values to the corresponding username and groupname.
+It does this by monitoring the `/etc/passwd` and `/etc/group` files on the host.
+This leads to the restrictions that user and groupnames inside the guest might not be correctly resolved.
```

Comment:
What are those restrictions? When doesn't it work? I think it's important to document it.

Reply:
I added an example. WDYT?

Reply:
Perhaps say that the event is always enriched with the user name in the host and that the container one isn't taken into consideration?

I think it can be something like:

> It does this by monitoring the `/etc/passwd` and `/etc/group` files on the host. It means the events are enriched with the user and group names from the host, so the ones for the container are ignored.